pkg/policies/controlplane/components/decider.go

```diff
@@ -63,20 +63,20 @@ func NewDeciderAndOptions(timedProto *policylangv1.Decider, _ int, policyReadAPI
 		return nil, fx.Options(), fmt.Errorf("unknown operator")
 	}
 	timed := &Decider{
```

Comment:
Can rename timed and `timedProto` variables to `decider` and `deciderProto` for clarity